app/models/service_template.rb

```diff
@@ -56,6 +57,7 @@ class ServiceTemplate < ApplicationRecord
   has_many   :dialogs, -> { distinct }, :through => :resource_actions
 
   has_many   :miq_requests, :as => :source, :dependent => :nullify
+  has_many   :active_requests, -> { where(:request_state => %w(active queued)) }, :as => :source, :class_name => "MiqRequest"
```

Comment:
@bzwei can you confirm this covers all "active requests"?

Reply:
You can use constants
https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_request.rb#L4

Reply:
:heart: perfect